src/rgw/rgw_civetweb_frontend.cc

```diff
@@ -35,13 +35,19 @@ int RGWMongooseFrontend::run() {
   char thread_pool_buf[32];
   snprintf(thread_pool_buf, sizeof(thread_pool_buf), "%d",
 	   (int)g_conf->rgw_thread_pool_size);
+
+  char req_timeout_buf[32];
+  snprintf(req_timeout_buf, sizeof(req_timeout_buf), "%d",
+	   (int)g_conf->rgw_thread_pool_size * 1000);
```

Comment:
Yup, it should. I overlooked that while editing. I'm pretty sure you saved me several hours on tomorrow's testing and investigation why the setting doesn't work. Thanks a lot! :-)